pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -200,17 +200,16 @@ func (p *staticPolicy) validateState(s state.State) error {
 			return fmt.Errorf("default cpuset cannot be empty")
 		}
 		// state is empty initialize
-		allCPUs := p.topology.CPUDetails.CPUs()
-		s.SetDefaultCPUSet(allCPUs)
+		s.SetDefaultCPUSet(p.allocatableCPUs())
```

Comment:
I'm still somehow bothered by the fact we initialize the state in a function called `validateState` but we did this before so I guess we can keep it this way and refactor later